### ISSUE TITLE
Remove "comply to WCAG" assumptions

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#549]
Several rules have assumptions like:

> The ARIA attribute is being used to comply to WCAG.

This is pretty vague, and I don't believe this helps readers understand why they might see false positives. My own rule of thumb is that any content author should be able to look at these assumptions and determine whether or not the result they saw is correct, or that one of the assumptions applied.

In this particular case, the assumption could be put like this:

> Attributes that start with `aria-` are used to set some state or property that should be picked up by assistive technologies. If any `aria-` attributes are not used for that purpose, the results of this rule may be incorrect.

**Comment by john-urbilog** [comment]
Hi Wilco ...

Can I make pull requests to the master or do I need to fork the project
first and then submit my pull-requests from there ?

John







*John HICKS*

Consultant en Accessibilité Senior

Docteur en Sciences Cognitives

06 28 06 58 60

*Urbilog*

31, rue Papin

59650 VILLENEUVE D'ASCQ

03 28 55 21 30

*Découvrez Urbilog sur notre site Internet **www.urbilog.fr
<http://www.urbilog.fr>*

*Expertise et accompagnement en accessibilité numérique*

*Rendez vos outils digitaux accessibles à tous !*









Le jeu. 23 mai 2019 à 10:20, Wilco Fiers <notifications@github.com> a
écrit :

**Comment by jkodu** [comment]
@john-urbilog You will have to fork the project and open a PR from there, with the concerned changes.

**Comment by annethyme** [comment]
The only published WAI-ARIA rules I could find were these: 

- [ARIA State or Property Allowed](https://act-rules.github.io/rules/5c01ea)
- [aria-hidden with focusable content](https://act-rules.github.io/rules/6cfa84)
- [Aria required context role](https://act-rules.github.io/rules/5f99a7)
- [Aria required owned element](https://act-rules.github.io/rules/6a7281)

None of those have the assumption anymore mentioned in this issue, and the two open PRs for WAI-ARIA related things have fuller assumptions, so I will close this issue.

**Comment by john-urbilog** [comment]
Hello,

I actually updated 5 rulles with fuller assumptions, but it seems only 1
found its way back to the review (?)

Trying to fix the requests for the other ones.... or is it cancelled now?

John






*John HICKS*

Consultant en Accessibilité Senior

Docteur en Sciences Cognitives

06 28 06 58 60

*Urbilog*

31, rue Papin

59650 VILLENEUVE D'ASCQ

03 28 55 21 30

*Découvrez Urbilog sur notre site Internet **www.urbilog.fr
<http://www.urbilog.fr>*

*Expertise et accompagnement en accessibilité numérique*

*Rendez vos outils digitaux accessibles à tous !*









Le ven. 19 juil. 2019 à 10:20, Anne Thyme <notifications@github.com> a
écrit :
